pkg/sql/schemachanger/scpb/elements.proto

```diff
@@ -304,10 +308,23 @@ message SchemaChild {
 
 message Sequence {
   uint32 sequence_id = 1 [(gogoproto.customname) = "SequenceID", (gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/sql/sem/catid.DescID"];
-
+  oneof optional_restart_with{
+    int64 restart_with = 2;
+  }
   bool is_temporary = 10;
 }
 
+message SequenceOptions {
+  uint32 sequence_id = 1 [(gogoproto.customname) = "SequenceID", (gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/sql/sem/catid.DescID"];
+  int64 increment = 2;
+  int64 max = 3;
+  int64 min = 4;
+  int64 start = 5;
+  int64 cache_size = 6;
+  bool virtual = 7;
+  string as_integer_type = 8;
+}
```

Comment:
This won't work when we have to implement `ALTER SEQUENCE`. How would you set targets for
```
BEGIN;
ALTER SEQUENCE seq MINVALUE 10;
ALTER SEQUENCE seq MAXVALUE 100;
COMMIT;
```

Instead, consider adding a `SequenceOption` element:
```
message SequenceOptions {
  uint32 sequence_id = 1 [(gogoproto.customname) = "SequenceID", (gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/sql/sem/catid.DescID"];
  string key = 2;
  string value = 3;
}
```
and bind the `DescID` attr to the `SequenceID` field and the `Name` attr to `Key` field.

Reply:
Good point, I'll refresh the code shortly with that.